dashboard/test/models/script_test.rb

```diff
@@ -1505,156 +1371,6 @@ def populate_cache_and_disconnect_db
-      lesson 'Lesson1', display_name: 'Lesson1'
-      level '#{l.name}'
-    UNIT
-    new_dsl = <<-UNIT
-      lesson 'Lesson1', display_name: 'Lesson1'
-      level '#{l.name}'
-    UNIT
-    unit_data, _ = ScriptDSL.parse(old_dsl, 'a filename')
-    unit = Script.add_unit(
-      {
-        name: 'challengeTestScript',
-        properties: Script.build_property_hash(unit_data)
-      },
-      unit_data[:lesson_groups]
-    )
-    assert unit.project_widget_visible
-
-    unit_data, _ = ScriptDSL.parse(new_dsl, 'a filename')
-    unit = Script.add_unit(
-      {
-        name: 'challengeTestScript',
-        properties: Script.build_property_hash(unit_data)
-      },
-      unit_data[:lesson_groups]
-    )
-
-    refute unit.project_widget_visible
-  end
-
-  test 'can unset the announcements attribute' do
-    l = create :level
-    old_dsl = <<-UNIT
-      announcements [{"notice"=>"notice1", "details"=>"details1", "link"=>"link1", "type"=>"information"}]
-      lesson 'Lesson1', display_name: 'Lesson1'
-      level '#{l.name}'
-    UNIT
-    new_dsl = <<-UNIT
-      lesson 'Lesson1', display_name: 'Lesson1'
-      level '#{l.name}'
-    UNIT
-    unit_data, _ = ScriptDSL.parse(old_dsl, 'a filename')
-    unit = Script.add_unit(
-      {
-        name: 'challengeTestScript',
-        properties: Script.build_property_hash(unit_data)
-      },
-      unit_data[:lesson_groups]
-    )
-    assert unit.announcements
-
-    unit_data, _ = ScriptDSL.parse(new_dsl, 'a filename')
-    unit = Script.add_unit(
-      {
-        name: 'challengeTestScript',
-        properties: Script.build_property_hash(unit_data)
-      },
-      unit_data[:lesson_groups]
-    )
-
-    refute unit.announcements
-  end
-
-  test 'can set custom curriculum path' do
-    l = create :level
-    dsl = <<-UNIT
-      curriculum_path '//example.com/{LOCALE}/foo/{LESSON}'
-      lesson 'Lesson1', display_name: 'Lesson1', has_lesson_plan: true
-      level '#{l.name}'
-      lesson 'Lesson2', display_name: 'Lesson2', has_lesson_plan: true
-      level '#{l.name}'
-    UNIT
-    unit_data, _ = ScriptDSL.parse(dsl, 'a filename')
-    unit = Script.add_unit(
-      {
-        name: 'curriculumTestScript',
-        properties: Script.build_property_hash(unit_data),
-      },
-      unit_data[:lesson_groups]
-    )
-    assert_equal CDO.curriculum_url('en-us', 'foo/1'), unit.lessons.first.lesson_plan_html_url
-    with_locale(:'it-IT') do
-      assert_equal CDO.curriculum_url('it-IT', 'foo/2'), unit.lessons.last.lesson_plan_html_url
-    end
-
-    unit.curriculum_path = '//example.com/foo/{LESSON}'
-    unit.save!
-    assert_equal '//example.com/foo/1', unit.lessons.first.lesson_plan_html_url
-    assert_equal '//example.com/foo/2', unit.lessons.last.lesson_plan_html_url
-
-    unit.curriculum_path = nil
-    unit.save!
-    assert_equal '//test.code.org/curriculum/curriculumTestScript/1/Teacher', unit.lessons.first.lesson_plan_html_url
-  end
```

Comment:
removed lines between 1279-1656 all look like defunct script seeding functionality to me.